plog/plog

```diff
@@ -459,6 +469,28 @@ main () {
     shift $(($OPTIND - 1))
 
 
+# Check compression type
+    if $(command -v  bzip2 >/dev/null) ; then
+        PLOG_COMPRESSED_EXT="${PLOG_COMPRESSED_EXT:-bz2}"
+    else
+        PLOG_COMPRESSED_EXT="${PLOG_COMPRESSED_EXT:-gz}"
+    fi
```

Comment:
I think should this be done before the `getopts` block to allow the `-g` flag to override the environment variable.

Reply:
Good point (I tried to follow your rule). The structure is a little different since `-g` is overriding any inherited value before we get here (maybe that is bad practice though), so it has the desired effect..